index.html

```diff
@@ -51,6 +51,123 @@
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">2015</div>
+                </div>
+                <div class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="work-stacks">
+                    <li class="stack"><a href="#" >HTML</a></li>
+                    <li class="stack"><a href="#" >Css</a></li>
+                    <li class="stack"><a href="#" >Javascript</a></li>
+                </ul>
+                <button class="work-link">See project</button>
+             </div>
+             <div class="work">
+                <img alt="work one" class="work-image" src="./assets/images/work2.svg"/>
+                <h2 class="work-title">Multi-Post <br> Stories</h2>
+                <div class="work-info">
+                    <div class="work-info-title">Canopy</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">Back End Dev</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">2015</div>
+                </div>
+                <div class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="work-stacks">
+                    <li class="stack"><a href="#" >HTML</a></li>
+                    <li class="stack"><a href="#" >Css</a></li>
+                    <li class="stack"><a href="#" >Javascript</a></li>
+                </ul>
+                <button class="work-link">See project</button>
+                
+             </div>
+             <div class="work">
+                <img alt="work one" class="work-image" src="./assets/images/works.svg"/>
+                <h2 class="work-title">Tonic</h2>
+                <div class="work-info">
+                    <div class="work-info-title">Canopy</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">Back End Dev</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">2015</div>
+                </div>
+                <div class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="work-stacks">
+                    <li class="stack"><a href="#" >HTML</a></li>
+                    <li class="stack"><a href="#" >Css</a></li>
+                    <li class="stack"><a href="#" >Javascript</a></li>
+                </ul>
+                <button class="work-link">See project</button>
+             </div>
+        </section>
+        <section class="about" id="about">
+            <p class="bio-header">
```

Comment:
- [ ] Kindly use a header tag hx, This way will be semantically correct and the screen readers.

Reply:
> - [ ] Good job so far👍 As requested by the previous reviewer. Please kindly use a heading tag, hx, instead of the p tag.  By using appropriate heading levels, you can help search engines understand the main topics and subtopics of a page, which can improve its ranking in search results.☺